netci.groovy

```diff
@@ -308,9 +308,10 @@ def targetGroupOsMap = ['netcoreapp': ['Windows 10', 'Windows 7', 'Windows_NT',
 
                 def newJob = job(Utilities.getFullJobName(project, newJobName, isPR)) {
                     // On Windows we use the packer to put together everything. On *nix we use tar
+                    def skipTests = targetGroup == 'netfx' ? '' : '-SkipTests';
                     steps {
                         batchFile("call \"C:\\Program Files (x86)\\Microsoft Visual Studio 14.0\\VC\\vcvarsall.bat\" x86 && build.cmd -${configurationGroup} -framework:${targetGroup}")
-                        batchFile("call \"C:\\Program Files (x86)\\Microsoft Visual Studio 14.0\\VC\\vcvarsall.bat\" x86 && build-tests.cmd -${configurationGroup} -framework:${targetGroup} -SkipTests")
```

Comment:
You're right, it would need to use targetGroupOsMap since netfx is not relevant to ubuntu, etc.
I don't see a good reason why the netfx tests aren't handled in this block (370+) though. It seems to give you some goodness like archiving results.
I'm OK though with doing it this way pending some cleanup